vita3k/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -83,9 +84,32 @@ EXPORT(int, _sceDisplayGetFrameBufInternal) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, _sceDisplayGetMaximumFrameBufResolution) {
-    TRACY_FUNC(_sceDisplayGetMaximumFrameBufResolution);
-    return UNIMPLEMENTED();
+EXPORT(SceInt32, _sceDisplayGetMaximumFrameBufResolution, SceInt32 *width, SceInt32 *height) {
+    TRACY_FUNC(_sceDisplayGetMaximumFrameBufResolution, width, height);
+    if (!width || !height)
+        return 0;
+    if (emuenv.cfg.pstv_mode) {
+        *width = 1920;
+        *height = 1088;
+    } else {
+        auto &title_id = emuenv.io.title_id;
+        bool cond = title_id == "PCSG80001"
+            || title_id == "PCSG80007"
+            || title_id == "PCSG00318"
+            || title_id == "PCSG00319"
+            || title_id == "PCSG00320"
+            || title_id == "PCSG00321"
+            || title_id == "PCSH00059";
```

Comment:
Might want to add a comment saying the PS Vita does this exact same check.

Reply:
Oups, eu yes sorry

Reply:
Done 👌